draft-ietf-quic-manageability.md

```diff
@@ -1055,16 +1055,17 @@ It is expected that any QoS handling in the network, e.g. based on use of
 DiffServ Code Points (DSCPs) {{?RFC2475}} as well as Equal-Cost
 Multi-Path (ECMP) routing, is applied on a per flow-basis (and not per-packet)
 and as such that all packets belonging to the same QUIC connection get uniform
-treatment. Using ECMP to distribute packets from a single flow across multiple
+treatment.
```

Comment:
I wonder if a reference to RFC7657 would make sense also here.

Reply:
PRs cheerfully accepted :)

Reply:
Actually, scratch that; I think the reference belongs in Applicability (PR #383).